src/algebra/category/CommRing/basic.lean

```diff
@@ -132,3 +132,68 @@ instance has_forget_to_CommSemiRing : has_forget₂ CommRing CommSemiRing :=
 has_forget₂.mk' (λ R : CommRing, CommSemiRing.of R) (λ R, rfl) (λ R₁ R₂ f, f) (by tidy)
 
 end CommRing
+
+
+namespace ring_equiv
+
+variables {X Y : Type u}
+
+section
+variables [ring X] [ring Y]
+
+/-- Build an isomorphism in the category `Ring` from a `ring_equiv` between `ring`s. -/
+def to_Ring_iso (e : X ≃+* Y) : Ring.of X ≅ Ring.of Y :=
+{ hom := e.to_ring_hom,
+  inv := e.symm.to_ring_hom }
+
+@[simp] lemma to_Ring_iso_hom {e : X ≃+* Y} : e.to_Ring_iso.hom = e.to_ring_hom := rfl
+@[simp] lemma to_Ring_iso_inv {e : X ≃+* Y} : e.to_Ring_iso.inv = e.symm.to_ring_hom := rfl
```

Comment:
We could use `@[simps]` for these, right? And then get rid of the surrounding `section`. Same for `comm_ring` below.

Reply:
Done!